resources.go

```diff
@@ -54,14 +54,24 @@ func (p *Package) Add(path string, file File) error {
 	return nil
 }
 
-// AddFile is a helper function that adds the files from the path into the
```

Comment:
Removing this is breaking change. Any reason why we can't keep this?

Reply:
I hadn't considered the function being called outside of go-resources.  I will add a new function instead.